### PR TITLE
Grammar and wording edits to ARD Expanded Processing Extent

### PR DESCRIPTION
**Pull request by benji-glitsos-ga**
* [x] I have spellchecked my content using Microsoft Word, Grammarly, or otherwise.
* [ ] If required, update the [DEA Tech Alerts and Changelog](TechAlertsChangelog) and the [DEA Sandbox Updates banner](SandboxUpdatesBanner).

[TechAlertsChangelog]: https://github.com/GeoscienceAustralia/dea-knowledge-hub/blob/main/docs/tech-alerts-changelog/_tech_alerts_changelog.md
[SandboxUpdatesBanner]: https://bitbucket.org/geoscienceaustralia/datakube-apps/src/64c28bbf3d0e019d8940547a22f78b9bfd58d739/clusters/dea-sandbox/sandbox.yaml

Please review the [diff of the changes I've made](https://github.com/GeoscienceAustralia/dea-knowledge-hub/pull/175/files).

### CODE REVIEW

**Comment by benji-glitsos-ga** [comment]
**Preview:** https://pr-175-preview.khpreview.dea.ga.gov.au/guides/reference/ard-expanded-processing-extent/

File: docs/guides/reference/ard-expanded-processing-extent.md [hunk 1]
```diff
@@ -2,17 +2,22 @@
 
 DEA produces surface reflectance products from Landsat and Sentinel-2 satellite observations which cover the continent and some near-shore islands of Australia. This data is used to produce derivative products such as DEA Coastlines, Water Observations, and Fractional Cover. 
 
-We are extending our baseline satellite data coverage to include Christmas Island, Cocos (Keeling) Islands, Coral Sea Islands, Heard and McDonald Islands, Norfolk Island, and islands in the Torres Strait. Additionally, we are processing imagery over shallow ocean areas to support bathymetry and oceanographic science.
+We are extending our baseline satellite data coverage to include Christmas Island, Cocos (Keeling) Islands, Coral Sea Islands, the Heard and McDonald Islands, Norfolk Island, and islands in the Torres Strait. Additionally, we are processing imagery over shallow ocean areas to support bathymetry and oceanographic science.
```
Comment (robbibt) [inline docs/guides/reference/ard-expanded-processing-extent.md:5]: It's the "Territory of Heard Island and McDonald Islands", so the original wording here is fine.

```suggestion
We are extending our baseline satellite data coverage to include Christmas Island, Cocos (Keeling) Islands, Coral Sea Islands, Heard and McDonald Islands, Norfolk Island, and islands of the Torres Strait. Additionally, we are processing imagery over shallow ocean areas to support bathymetry and oceanographic science.

```

File: docs/guides/reference/ard-expanded-processing-extent.md [hunk 2]
```diff
@@ -47,17 +52,17 @@ This table contains the full list of locations, including the additional locatio
...
+* - Daru, Western Province Papua New Guinea, and Northern Warrior Reefs
   - 098-066, 54LZR
 * - Elizabeth Reef
   - 085-081, 57JVG, 57JWG 
 * - Middleton Reef
   - 085-080, 57JVH, 57JWH 
 * - Frederick Reef, Queensland
   - 088-075, 56KQB, 57KTS, 56KQA 
-* - Coral Sea – Scenes with Ocean depth less than 200m
+* - Coral Sea (scenes with Ocean depth less than 200 m)
```
Comment (robbibt) [inline docs/guides/reference/ard-expanded-processing-extent.md:65]: @OHehir-Aaron  Can we remove the reference to 200 m here? It was a very informal rule and not used to actually select the scenes (that was done via assessing Australia's territorial sea boundaries I think)

File: docs/guides/reference/ard-expanded-processing-extent.md [hunk 3]
```diff
@@ -89,33 +94,32 @@ dc.find_datasets(product="ga_ls8c_ard_3", limit=10, final_ancillaries="nonstanda
...
+[BRDF](/guides/about/glossary/#brdf) is a solar illumination correction that adjusts reflectance intensity in differing amounts in differing directions. The light that comes from the Sun reflects off the Earth and is received by the sensor is adjusted based on the viewing or solar angle geometries of MODIS data.
 
-Sometimes over offshore locations we have found BRDF values to be unphysical. We have implemented a BRDF solution over the offshore locations, which masks out data from solar illumination correction where Fvol/Fiso or Fgeo/Fiso > 1. This ensures that we do not violate the theoretical definition of these parameters, meaning the values are realistic.
+Sometimes over offshore locations, we have found BRDF values to be unphysical. We have implemented a BRDF solution over the offshore locations, which masks out data from solar illumination correction where $\frac{Fvol}{Fisor} > 1$ or $\frac{Fgeo}{Fiso} > 1$. This ensures that we do not violate the theoretical definition of these parameters, meaning the values are realistic.
```
Comment (robbibt) [inline docs/guides/reference/ard-expanded-processing-extent.md:113]: @OHehir-Aaron Can we please explain what "unphysical" means here for the benefit of our users who won't be familiar with the term? Or altneratively, add it to the Knowledge Hub glossary.

File: docs/guides/reference/ard-expanded-processing-extent.md (hunk 1 above)
Comment (OHehir-Aaron) [reply docs/guides/reference/ard-expanded-processing-extent.md:5]: Thanks Robbi, I really appreciate the second set of eyes over this document.

File: docs/guides/reference/ard-expanded-processing-extent.md (hunk 2 above)
Comment (OHehir-Aaron) [reply docs/guides/reference/ard-expanded-processing-extent.md:65]: Sure!

**Comment by robbibt** [comment]
Hey @benji-glitsos-ga and @OHehir-Aaron, apologies for being annoyingly pedantic about names here, but I think it's important given this new data will become part of Australia's "official" data holdings for our external territories and the maps in this article might get shared wider than this doc. 

Can we please update the two maps here to use the official territory names, i.e.:

- "Cocos Island" > "Cocos **(Keeling)** Island**s**"
- "The Heard and MacDonald Islands" > "Heard **Island** and McDonald Islands"

image

Can double check against the lists here: https://en.wikipedia.org/wiki/States_and_territories_of_Australia

File: docs/guides/reference/ard-expanded-processing-extent.md [hunk 4]
```diff
@@ -47,17 +52,17 @@ This table contains the full list of locations, including the additional locatio
   - 122-067, 123-067, 48LWP, 48LXP 
 * - Cocos (Keeling) Island
```
Comment (robbibt) [inline docs/guides/reference/ard-expanded-processing-extent.md:53]:
```suggestion
* - Cocos (Keeling) Islands

```

**Review by CEKrause — APPROVED** [review]
Nice changes. Thanks.

**Pull request merged**